packages/react/src/components/ComboBox/ComboBox-story.js

```diff
@@ -120,4 +155,9 @@ storiesOf('ComboBox', module)
         text: `Sometimes you want to perform an async action to trigger a backend call on input change.`,
       },
     }
-  );
+  )
+  .add('fully controlled', () => <ControlledComboBoxApp {...props()} />, {
```

Comment:
Probably "application-level control for selection"? Not sure if our combo box allows "fully-controlled" scenario for every key props.